TabularClassifier.ipynb

```diff
@@ -0,0 +1,207 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tackling Tabular Data with BOLT\n",
+    "\n",
+    "Tabular data is some of the most common data in real world applications. Consisting of mostly categories and numbers, tabular data is used in a variety of contexts including financial services, fraud detection, healthcare, and more. For these problems, gradient boosted decision trees like XGBoost have long been the algorithms of choice for their ability to quickly learn efficient representational decision boundaries on the data. However, deep learning has made significant strides in this area with developments such as TabNet and even our very own BOLT engine.\n",
+    "\n",
+    "Our deep learning engine BOLT is an efficient neural framework that has proven effective on a wide array of problems. Most recently, initial experiments have shown that BOLT can beat top methods such as XGBoost and TabNet on several benchmark tabular datasets. Additionally, BOLT has the added capability that it works exceptionally well as an out of the box classifier, requiring much less effort to train than other methods."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Try out the BOLT TabularClassifier\n",
+    "\n",
+    "### Download and Set Up the Census Income Dataset"
```

Comment:
Can you add a part here introducing the demo? It's very abrupt to go from a high level overview right into code. e.g. explain the dataset, that we're doing this demo but you can easily substitute for your own data, etc.

Reply:
Also downloading should probably be "step 1"